privacyidea/lib/smsprovider/FirebaseProvider.py

```diff
@@ -98,6 +99,28 @@ def submit_message(self, firebase_token, data):
 
         return res
 
+    def check_configuration(self):
+        """
+        This method checks the sanity of the configuration of this provider.
+        If there is a configuration error, than an exception is raised.
+        :return:
+        """
+        json_file = self.smsgateway.option_dict.get(FIREBASE_CONFIG.JSON_CONFIG)
```

Comment:
Should we check if the configuration file exists?

Reply:
isn't it (implicitly) checked in the `with open(...` part?

Reply:
Yes, If the file does not exist, imho an exception is raised and we get an error anyways.
Would work like expected.

Reply:
OK, yes, we get a HTTP 500 if the file does not exist. The ``SMSError`` exceptions below also create HTTP 500 responses, so we can keep it that way.